verifiers/internal/gha/npm_sigstore_tuf.go

```diff
@@ -0,0 +1,104 @@
+package gha
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	sigstoreTuf "github.com/sigstore/sigstore-go/pkg/tuf"
+)
+
+const (
+	attestationKeyUsage = "npm:attestations"
+	attestationKeyID    = "SHA256:jl3bwswu80PjjokCgh0o2w5c2U4LhQAE57gj9cz1kzA"
+	targetPath          = "registry.npmjs.org/keys.json"
+)
+
+var (
+	ErrorMissingNpmjsKeyIDKeyUsage = errors.New("could not find a key with the specified 'keyId' and 'keyUsage'")
+	ErrorCouldNotFindTarget        = errors.New("could not get the target from the tuf root")
+	ErrorCouldNotParseKeys         = errors.New("could not parse keys file content")
```

Comment:
Do we use this outside the package or just for tests? If the latter, we could make this unexported.
```suggestion
	errMissingNpmjsKeyIDKeyUsage = errors.New("could not find a key with the specified 'keyId' and 'keyUsage'")
	errCouldNotFindTarget        = errors.New("could not get the target from the tuf root")
	errCouldNotParseKeys         = errors.New("could not parse keys file content")
```

Reply:
changed